### DISCUSSION TITLE
Jsdelivr link for tailwind

### DISCUSSION THREAD

**Discussion by solomonshalom** [tailwindlabs/tailwindcss#6172] [Help]
Hey Devs,

It would be great if I would get the jsdelivr link for tailwind since I didn't want to install it via NPM.  Thank You

**Comment by NovusOpus** [comment]
### There's a new cool feature coming in **Tailwind CSS V3.0** That you can try now! for testing and development purposes.

## **The Just in Time CDN**

###  Quote from the [Tailwind CSS V3.0 Announcement](https://github.com/tailwindlabs/tailwindcss/discussions/5668):

> ## Just-in-Time CDN
> Something that has always sucked about our CDN builds is that we always had to disable tons of awesome features to keep the file size down to something the browser was willing to parse.
> 
> For Tailwind CSS v3.0, we're doing it differently. Inspired by some awesome work Marcel Pociot did on tailwindcss-jit-cdn, we've built a JS library that compiles Tailwind in the browser, and uses MutationObserver to keep track of all of the classes you're using.
> 
> It lets you use every feature Tailwind CSS has to offer, and does it in 93kB — which ain't bad for something originally designed to run in node with no file size considerations at all.
> 
> ### **The Just-in-Time CDN is intended for development purposes only, and should not be used in production.**
> 
> <details><summary>Why shouldn't it be used in production? (Click to show content)</summary>
> <p>
> 
> The biggest reason is because it uses MutationObserver to add the styles, it can't detect styles for dynamically created elements fast enough to avoid a flash of unstyled content (FOUC).
> 
> For example, say you have some JavaScript that opens a modal, and the modal is supposed to transition in when it opens. When the modal opens, the HTML for it is inserted into the DOM right away, but the styles might not exist for it yet because you haven't used those same classes elsewhere in the file. The observer will fire, and Tailwind will generate the styles, but the modal is already open, so you're going to see an unstyled flash the first time it opens.
> 
> We recommend pulling in the JIT CDN as a blocking (so not deferred) script to avoid the FOUC for the very initial render, but that of course means it adds 100ms (or whatever) before the page is even rendered. Not a big deal really but using a static CSS file is way faster.
> 
> It's also quite large (almost 100kB compressed) whereas compiling your CSS ahead of time usually leads to something closer to 10kb compressed, and with no run time overhead.
> 
> TLDR; It's probably fine for simple static pages but it's really much better to build the static CSS file.
> 
> </p>
> </details>
> 
> To try it out, throw this `<script>` tag in your `<head>`:
> ```html
> <script src="https://cdn-tailwindcss.vercel.app/"></script>
> ```
> Like this:
> ```diff
>   <!DOCTYPE html>
>   <html lang="en">
>     <head>
>       <meta charset="utf-8">
>       <meta name="viewport" content="width=device-width, initial-scale=1.0" />
>       <title>Example</title>
> +     <script src="https://cdn-tailwindcss.vercel.app/"></script>
>     </head>
>     <body>
>       
>     </body>
>   </html>
> ```
> We'll give it a more permanent URL in the future but this is good enough for the alpha.
> 
> ### Using first-party plugins
> First-party plugins can be automatically loaded using the plugins query parameter:
> 
> ```html
> <script src="https://cdn-tailwindcss.vercel.app/?plugins=forms,typography,aspect-ratio,line-clamp"></script>
> ```
> The `plugins` parameter accepts a comma-separated list of plugins, and you can optionally specifiy a version number or range for each plugin, for example `?plugins=forms@0.3.4,typography@^0.5`
> 
> ### Customizing your config
> To customize your Tailwind CSS configuration, override `tailwind.config`:
> 
> ```HTML
> <!DOCTYPE html>
> <html lang="en">
>   <head>
>     <meta charset="utf-8" />
>     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
>     <title>Example</title>
>     <script src="https://cdn-tailwindcss.vercel.app/"></script>
>     <script>
>       tailwind.config = {
>         theme: {
>           extend: {
>             colors: {
>               tomato: 'tomato',
>             },
>           },
>         },
>       }
>     </script>
>   </head>
>   <body>
>     
>   </body>
> </html>
> ```
> ### Adding custom CSS
> Use `<style type="text/tailwindcss">` elements to add any custom CSS that that you'd like to process with Tailwind:
> 
> ```HTML
> <!DOCTYPE html>
> <html lang="en">
>   <head>
>     <meta charset="utf-8" />
>     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
>     <title>Example</title>
>     <script src="https://cdn-tailwindcss.vercel.app/"></script>
>     <style type="text/tailwindcss">
>       body {
>         @apply bg-red-500;
>       }
>     </style>
>   </head>
>   <body>
>     
>   </body>
> </html>
> ```
